tests/fixtures/schemas/generate_schema.py

```diff
@@ -27,7 +27,7 @@ def main():
     args = parser.parse_args()
 
     sg_url = args.shotgun
-    sg = Shotgun(sg_url, login=raw_input("Login: "), password=getpass.getpass())
+    sg = Shotgun(sg_url, login=input("Login: "), password=getpass.getpass())
```

Comment:
Is it clear in the documentation that we don't support Python 2 anymore?

Reply:
I'm just updating the documentation. I just found out that the [official Jira client](https://github.com/pycontribs/jira) supports only Python 3.8+ so I can't include CI jobs for 3.7.

Reply:
Thanks to the newly added CI makes me realize this.